source/code/plugin/containerlogtailfilereader.rb

```diff
@@ -76,14 +76,15 @@ def initialize(path, pe, read_from_head, log, &receive_lines)
         @receive_lines = receive_lines
         @rotate_handler = RotateHandler.new(path, log, &method(:on_rotate))
         @io_handler = nil
+        @containerIDFilePath = "/var/opt/microsoft/docker-cimprov/state/ContainerInventory/"
       end
 
       attr_reader :path
 
       def wrap_receive_lines(lines)
 	newLines = []
 	containerID = @path.split('/').last.chomp('-json.log')
-	containerInspectInformation = "/var/opt/microsoft/docker-cimprov/state/ContainerInventory/" + containerID
+	containerInspectInformation = @containerIDFilePath + containerID
```

Comment:
Can you add comments describing the docker log file location . Just to make it more readable and easy to maintain ?